Resources/public/js/views/services/ez-contenteditviewservice.js

```diff
@@ -282,7 +282,16 @@ YUI.add('ez-contenteditviewservice', function (Y) {
          * @param {Function} callback
          */
         _loadOwner: function (id, callback) {
-            this._loadModel('owner', id, {}, "Could not load the user with id '" + id + "'", callback);
+            var model = this.get('owner'),
+                loadOptions = {api: this.get('capi')};
+
+            model.set('id', id);
+            model.load(loadOptions, Y.bind(function (error) {
+                if (error) {
+                   this.set('owner', null);
```

Comment:
about that, given the view service are kept in memory, won't this break any following usage of the content edit view service ?